DigitalLibrary/DigitalLibrary/MainScreens/BookDetailsScreen/BookDetailsView.swift

```diff
@@ -12,14 +12,21 @@ struct BookDetailsView: View {
 
     var body: some View {
         ScrollView {
-            Image(viewModel.photo)
-                .resizable()
-                .scaledToFit()
-                .cornerRadius(12)
-                .shadow(radius: 5)
-                .padding(.horizontal, 80)
-                .padding(.vertical, 25)
-            
+            if let photo = viewModel.photo,
+               let image = UIImage(data: photo) {
+                Image(uiImage: image)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 70, height: 90)
+                    .cornerRadius(8)
+            } else {
+                Image("default_book_cover")
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 70, height: 90)
+                    .cornerRadius(8)
+            }
```

Comment:
Why are you changing the properties of the image. Here the image should have different from the image at the cell.
```suggestion
                Image(uiImage: image)
                .resizable()
                .scaledToFit()
                .cornerRadius(12)
                .shadow(radius: 5)
                .padding(.horizontal, 80)
                .padding(.vertical, 25)
            } else {
                Image("default_book_cover")
                .resizable()
                .scaledToFit()
                .cornerRadius(12)
                .shadow(radius: 5)
                .padding(.horizontal, 80)
                .padding(.vertical, 25)
            }
```